vue.config.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
   configureWebpack: {
     resolve: {
       alias: {
-        'util': path.resolve(__dirname, 'src/util')
+        '@util': path.resolve(__dirname, 'src/util')
```

Comment:
👍 

It will be better to name an alias with prefix `@` if we should add new ones in the future, you know.